packages/dds/tree/src/feature-libraries/forest-summary/incrementalSummaryBuilder.ts

```diff
@@ -61,6 +61,23 @@ export const ForestSummaryTrackingState = {
 export type ForestSummaryTrackingState =
 	(typeof ForestSummaryTrackingState)[keyof typeof ForestSummaryTrackingState];
 
+/**
+ * The properties of a chunk tracked during the loading process.
+ * These are used to identify a chunk when it is decoded and recreate the tracking state
+ * as it was when the summary that the client is loading from was generated.
+ */
+interface ChunkLoadProperties {
+	/**
+	 * The encoded contents of the chunk.
+	 */
+	contents: EncodedFieldBatch;
+	/**
+	 * The path for this chunk's summary in the summary tree relative to the forest's summary tree.
+	 * This path is used to generate a summary handle for the chunk if it doesn't change between summaries.
+	 */
+	summaryPath: string;
```

Comment:
From the descriptions, I'm guessing these should be readonly:

```suggestion
	readonly contents: EncodedFieldBatch;
	/**
	 * The path for this chunk's summary in the summary tree relative to the forest's summary tree.
	 * This path is used to generate a summary handle for the chunk if it doesn't change between summaries.
	 */
	readonly summaryPath: string;
```